worlds/wargroove2/Options.py

```diff
@@ -0,0 +1,68 @@
+import typing
+from dataclasses import dataclass
+
+from Options import Choice, Range, DeathLink, PerGameCommonOptions, StartInventoryPool
+
+
+class IncomeBoost(Range):
+    """How much extra income the player gets per turn per boost received."""
+    display_name = "Income Boost"
+    range_start = 0
+    range_end = 100
+    default = 25
+
+
+class CommanderDefenseBoost(Range):
+    """How much extra defense the player's commander gets per boost received."""
+    display_name = "Commander Defense Boost"
+    range_start = 0
+    range_end = 8
+    default = 2
+
+
+class GrooveBoost(Range):
+    """How much extra groove the player's commander gets per boost received."""
+    display_name = "Groove Boost"
+    range_start = 0
+    range_end = 10
+    default = 3
+
+
+class LevelShuffleSeed(Range):
+    """What seed to use for level shuffling. 0 uses the multiworld seed."""
+    display_name = "Level Shuffle Seed"
+    range_start = 0
+    range_end = 0xFFFFFFFF
+    default = 0
+
+
+class CommanderChoice(Choice):
+    """How the player's commander is selected for missions.
+    Locked Random: The player's commander is randomly predetermined for each level.
+    Unlockable Factions: The player starts with Mercival and can unlock playable factions.
+    Random Starting Faction:  The player starts with a random starting faction and can unlock the rest.
+    When playing with unlockable factions, faction items are added to the pool."""
```

Comment:
Totally up to you, but you could change this to something like this. To me, this greatly improves the readability of it on WebHost:
```suggestion
    """How the player's commander is selected for missions.
    
    - Locked Random: The player's commander is randomly predetermined for each level.
    
    - Unlockable Factions: The player starts with Mercival and can unlock playable factions.
    
    - Random Starting Faction:  The player starts with a random starting faction and can unlock the rest.
    
    When playing with unlockable factions, faction items are added to the pool."""
```